ui/src/components/search/Search.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { css } from '@emotion/react';
 import Component from 'react-component-component';
```

Comment:
yuck! sorry you're having to deal with this library

Reply:
This project is from a whole different era, it's great